unit_tests/kernels/UnitTestKernelUtils.C

```diff
@@ -158,6 +158,56 @@ struct TrigFieldFunction
                              std::sin(a * pi * z));
   }
 
+  void gamint(const double* coords, double* qField) const //added: transition
```

Comment:
can you make the name more verbose? We have a lot of "gam/gamma" in the code in general. Maybe `gamma_intermittency"?

Reply:
That sounds good to me. I agree that there are so many gam/gamma in the code haha

Reply:
made changes to gamma_intermittency